### DISCUSSION TITLE
SOC Daly not rising while loading

### DISCUSSION THREAD

**Discussion by exenbema** [Louisvdw/dbus-serialbattery#431] [General]
Hi
with the 014 Driver i have a new issue in a new installation (all new firmwares are installed )
the soc of daly is decreasing while discharging, but its not rising while charging

![serial](https://user-images.githubusercontent.com/114774449/214229171-b2005e9d-fd3b-4941-9e18-0c2d8c7c6d7c.jpg)
it the utils file the only enabled management is the voltage (because daly and soc is never accurate)
- the value of the shunt is rising, so its ok
- the value in serialbattery details of available AH is not rising too
- the communication is ok, because i see he changes permantly the values of cell voltages...so this connection cant be the problem

do you have an idea?
thx markus

**Comment by exenbema** [comment]
i upgrade the bms to [BmsGD32E230_309_20220224_CADC.zip](https://www.microcharge.de/forum/index.php?attachment/995-bmsgd32e230-309-20220224-cadc-zip/) now its better its counting also while loading, but not quite ok, because daly counts the current false. there we need a current kalib - but im not sure how to do it, because the PCmaster tool was not good for that. i made the kalib with that but no difference then....

**Reply by janekx** [reply]
hi where did yuo got FW ?
